Keas.Core/Domain/Space.cs

```diff
@@ -5,6 +5,7 @@ namespace Keas.Core.Domain
 {
     // Facilities link deptSpace, flattened
     // ties together rooms and departments
+    [Table("Spaces")]
```

Comment:
@jSylvestre real table is singular now right?

Reply:
@srkirkland Correct. Table with the data is "Space"

Reply:
That's what I put!